client/src/components/Layout/Navbar/index.tsx

```diff
@@ -1,11 +1,20 @@
-import React from 'react';
+import React, { FC } from 'react';
+import { Navigation, HamburgerMenu } from "./styles";
+
+interface Props {
+  visible: boolean
+  setVisible: (visible: boolean) => void
+}
+
+const Navbar:FC<Props> = ({visible, setVisible}) => {
 
-const Navbar = () => {
   return (
-    <div>
-      Navigation
-    </div>
+    <Navigation>
+      <HamburgerMenu onClick={() => setVisible(!visible)}>
```

Comment:
hamburger menu should setting visibility to true and the other icon like X to false